zenodo/modules/spam/views.py

```diff
@@ -101,3 +104,36 @@ def delete(user_id):
         records = islice(rs.scan(), 10)
         ctx.update(records=records)
         return render_template('zenodo_spam/delete.html', **ctx)
+
+@blueprint.route('/<int:user_id>/safelist', methods=['POST'])
+@login_required
+def safelist_add_remove(user_id):
+    """Add or remove user from spam safelist."""
+    # Only admin can access this view
+    if not Permission(ActionNeed('admin-access')).can():
+        abort(403)
+
+    user = User.query.get(user_id)
+    if request.form['_method'] == 'post':
+        # Create safelist entry
+        SafelistEntry.create(user_id=user.id, notes='Added by {} ({})'.format(
```

Comment:
```suggestion
        SafelistEntry.create(user_id=user.id, notes=u'Added by {} ({})'.format(
```
Since we're in Python 2 and you never know :)